index.html

```diff
@@ -172,6 +172,34 @@ <h2 class="about-header">About me</h2>
             </div>
         </div> 
         <!-- work section ends-->
+        <div class="form-start">
+            <h2 class="form-header">I'm always interested in hearing about your new projects, so if you'd like to chat please get in</h2>
+            <form action="https://formspree.io/f/xlekbqne" method="post">
+                <fieldset>
+              <label for="full-name"></label>
+              <input type="text" id="full-name" name="full-name" maxlength="30" placeholder="Full name" required>
+              <input type="text" id="First-name" name="first-name" maxlength="" placeholder="First Name" required>
+              <input type="text" id="last-name" name="last-name" maxlength="" placeholder="Last Name" required>
+              <label for="email"></label>
+              <input type="email" id="email" name="email" placeholder="Email address" required>
+              <label for="message"></label>
+              <textarea id="message" name="message"  required>Enter text here</textarea>
+                    <button type="submit" class="submit">Get in touch</button>
+                </fieldset>
+            </form>
```

Comment:
- According to the project requirements, the contact form should aligned using the `flex` design attriute, please fix this 🙏